### ISSUE TITLE
Sickbeard Custom 20180629-7 version asking for username/password

### ISSUE THREAD

**Issue by kneighbour** [SynoCommunity/spksrc#3369]
_For new Package Requests, see the guidelines_  

### Setup
_Package Name:_Sickbeard Custom
_Package Version:_20180629-7

_NAS Model:_DS1812+
_NAS Architecture:_INTEL Atom D2700
_DSM version:_DSM 6.2-23739 Update 1

### Expected behavior
Sickbeard should just load without requesting a username and password. The previous version I had installed worked just fine. ie an hour ago!

### Actual behavior
asks for username and password. No idea what it is as I never had one setup.

### Steps to reproduce
_1._open URL http://10.1.1.30:8083
_2._
_3._

#### Package log
_Check Package Center or `/usr/local/{package}/var/`_
```
Insert the package log here
```
#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
Insert log here
```

**Comment by ymartin59** [comment]
May you please refer which git fork you have chosen at installation?

**Comment by kneighbour** [comment]
No option for the fork was given. Within the NAS/Synology software, a message came up saying there was an update. I clicked on the button to "Update" and that was it.

I tried to roll back to the older version (I downloaded the 2014 version), but the system will not allow the earlier version to install. I guess I would have to uninstall the package, then install the earlier version. That is a fair bit of work, so bit loathe to do that.

**Comment by kantjer** [comment]
If it's the SB login it's asking for you can retrieve that from

@appstore/sickbeard-custom/var/config.ini

**Comment by kneighbour** [comment]
Can I use the File Station to access that path?  I don't see anything like that, so I presume not.

**Comment by kneighbour** [comment]
Ok, made a bit of an effort and managed to get on via Telnet. I found the config file and I edited it with Vi. I did a :w and saved the changes. I loaded the file after saving it and checked that the changes were there. I then went into Diskstation and restarted the SB package. It did not work. Checking the config file again, the username and password were restored. Bit weird as I had made them blank ("") and had checked that the file had been updated.

I logged on with Putty as the root user.
